style/modules/box.css

```diff
@@ -0,0 +1,13 @@
+.box{
```

Comment:
there's some indentation stuff to clean up on this one

also, one more thing for the behavior -- let's have a media query that makes them 4px/4px (instead of 6) on mobile. I'm not sure exactly what we want for a breakpoint, but I had advised Ernest on his PR to use 600px for now.

It's worth your time to skim this -- https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Media_queries

but the short version is that you'll want something like 

```css
@media only screen and (max-width: 600px) {
  .box-red {
    ...
  }
}
```

that's hopefully the right code but I haven't tested it FYI

Reply:
also nice job on the SMACSS variant naming